### ISSUE TITLE
Release tarball changes after realease

### ISSUE THREAD

**Issue by krocard** [01org/parameter-framework#372]
# TLDR
Release tarball/zip content (thus hash) may change over time !

# Context
All (sufficiently recent) release tarballs contain a `.version` file that is expected to contain the tag of the release.
The tag name is added in the `.version` file during `git export`. The source controled `.version` contains the `%D` placeholder that is replaced with the commit ref names (not only tag, see git help `gitattributes`).

# Problem:
 - If no patch as been merged after a release is taged, the `.version` contains the expected tag AND `master`
 - After at least one patch is merged, the `.version` contains only the expected tag

Thus release tarballs are **not** immutable with current release process.

**Comment by krocard** [comment]
This bug is very inconvenient for Yocto integration as reported by @mgaio (who find the bug). Yocto expect tarball hash to be immutable.

**Comment by dawagner** [comment]
I can think of two solutions from the top of my head:
- Manually create release commits and manually update the version number. In order to reduce errors, we can write a release script.
- Manually generate a tarball and attach it to the release.

**Comment by krocard** [comment]
:+1: for the `git note` solution.

**Comment by shivank01** [comment]
I want to work on this bug. Is anyone already working on it???

**Comment by dawagner** [comment]
Hi, @shivank01. We are not currently working on it and you are welcome to!

**Comment by shivank01** [comment]
I am anoobie and i am working on this bug but unable to install asio in ubuntu.Can you please tell me how to install it on ubuntu.

**Comment by mgaio** [comment]
Hi, deb package is libasio-dev
